### PR TITLE
New code for Stand-Alone Agent with System Integration

### PR DESCRIPTION
**Pull request by IgorGuz2000**
## What does this PR do?
Added new scenarios for Stand-Alone Agent with System integration check

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests (`make unit-test`), and they are passing locally
- [x] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by mdelapenya — COMMENTED** [review]
Great contribution Igor!

I'll approve it once the Gherkin lint pass. You can run `make -C e2e lint` to verify it locally

**Review by adam-stokes — APPROVED** [review]

File: e2e/_suites/fleet/features/stand_alone_agent.feature [hunk 1]
```diff
@@ -70,3 +70,219 @@ Examples: default
...
+#@ubi8
+#@skip:arm64
+#Examples: Ubi8
+#| image |
+#| ubi8  |
+#
+#@default
+#Examples: default
+#| image   |
+#| default |
```
Comment (mdelapenya) [inline e2e/_suites/fleet/features/stand_alone_agent.feature:288]: If we are not going to use it, I'd directly remove it. Wdyt?

**Review by mdelapenya — APPROVED** [review]
LGTM! The power of BDD is adding more tests with no code! 👏

File: e2e/_suites/fleet/features/stand_alone_agent.feature (hunk 1 above)
Comment (IgorGuz2000) [reply e2e/_suites/fleet/features/stand_alone_agent.feature]: Done

**Comment by mdelapenya** [comment]
I think this is ready to go, let's merge it once the CI passes. 

Great job @IgorGuz2000 !

**Pull request merged**